packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Cross-Site Cookie Blocking
+---
+
+# Cross-Site Cookie Blocking
+
+Cross-site cookie blocking is a privacy-protection feature that is being added to web browsers like Safari and Firefox. It's likely to become common across most browsers in the future. It can disrupt certain customer-hosted flows in Okta.
+
+## Description of the problem
+
+Beginning from version 13.1, Safari [blocks third-party cookies by default](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), disrupting Okta functionality in certain flows. Firefox and other browsers are in the process of rolling out a similar change.
+
+This issue will not affect your organization if you use the Okta-hosted sign-in page. Only customers who host their own sign-in functionality can be affected.
+
+When customer-hosted code makes a call to Okta that relies on an [Okta Session Cookie](/docs/guides/session-cookie/) being included in the HTTP request, the browser blocks the cookie from reaching Okta because the user is on the customer domain and the cookie is being sent to Okta, a different domain.
+
+The specific functionality that is affected is session management, as well as token renewal in the OAuth 2.0 implicit flow.
+
+### Session management in customer-built applications
+
+If you host your own sign-in page, and use a self-hosted instance of the Okta Sign-In Widget to sign users in, or rely on JavaScript running in the user’s browser to make calls to Okta to handle session management, functionality can be broken by the browser’s blocking of third-party cookies.
+
+Okta session cookies that accompany XHR calls to Okta API endpoints like `/sessions/me` and `/user/me`	 are blocked by the browser because they are sent to a different domain than the one the user is on. Okta session cookies never get through to Okta, and Okta cannot complete the request.
+
+The result is that Okta returns 403 forbidden errors to the user or that your application repeatedly directs users back to the sign-in page.
+
+Specifically, this problem affects [certain methods](https://github.com/okta/okta-auth-js#third-party-cookies) of the Okta Auth JavaScript SDK. The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#okta-sign-in-widget) uses the affected Okta Auth JavaScript SDK methods internally. Any customer-developed code that directly makes XHR calls to the Okta [Sessions API](/docs/reference/api/sessions/) is also affected. 
+
+### Token renewal in customer-built SPAs that use the OAuth 2.0 implicit flow
+
+If your code uses the OAuth 2.0 implicit flow to handle token renewal (which typically occurs in the context of a SPA that uses the [implicit flow](/docs/guides/implement-implicit/)), the user's browser can block Okta session cookies from being sent, so that token renewal is never successfully completed.
```

Comment:
The issue is also seen with PKCE flow. The issue is caused by cookies not available in iframes of 3rd party domains. `token.getWithoutPrompt` opens an iframe to `/authorize` which loads an HTML page. If the Okta session cookie cannot be read by that page it will print an error "the user specified not to prompt but the user is not logged in" and return a failure.

Reply:
Changed to say that PKCE flow, in addition to implicit flow, is affected for token renewal.